libs-rs/mintable-fungible-token/src/lib.rs

```diff
@@ -590,6 +601,9 @@ mod tests {
     fn carol() -> AccountId {
         "carol.near".to_string()
     }
+    fn rainbow_bridge_eth_on_near_prover() -> AccountId {
```

Comment:
What is the purpose of this method?

Reply:
want to use this account, so follow the pattern of `alice()` `carol()` etc